app/views/shared/_navbar.html.erb

```diff
@@ -10,25 +10,17 @@
             <%= current_user.email %> <span class="caret"></span>
           </button>
           <ul class="dropdown-menu dropdown-menu-right">
-            <li>
-              <%= link_to 'Your Profile', user_path(current_user.id) %>
-            </li>
-            <li>
-              <%= link_to 'Dashboard', user_path(current_user.id) %>
-            </li>
+            <% if controller_name == "learn" %>
+              <li><%= link_to 'Pause Session', "/users/#{@user.id}/#{@deck.id}/preview" %></li>
+            <% end %>
```

Comment:
so pausing takes the user back to the preview, which has the link to begin/resume. so that makes sense. its just that theres no functionality tied to this button, say pausing a timer or something. additionally, the begin/resume button on the preview page links to `learn#index` which recreates the series.

so theres actually unexpected behavior around the pause button in that in order to get back to the same series you currently have to click the back button on the browser, which isnt great.

im suggesting we leave this out until we think it through more and are ready to build the necessary functionality around it. what do you think?